Platform.PCL/Realm.PCL/RealmObjectPCL.cs

```diff
@@ -334,5 +335,11 @@ public override bool Equals(object obj)
             RealmPCLHelpers.ThrowProxyShouldNeverBeUsed();
             return false;
         }
+
+        public TypeInfo GetTypeInfo()
```

Comment:
Can we make this an explicitly-implemented interface member so it doesn't show up in IntelliSense?